backend/user_service/app.js

```diff
@@ -7,7 +7,11 @@ const app = express();
 
 // Allow requests from 'https://question-service-1079323726684.asia-southeast1.run.app' (cloudrun question_service) with credentials
 app.use(cors({
-    origin: ['https://frontend-1079323726684.asia-southeast1.run.app', 'http://localhost:3000', 'http://localhost:5173'],
+    origin: ['https://frontend-1079323726684.asia-southeast1.run.app',
+      'http://localhost:3000',
+      'http://localhost:5173',
+      'https://question-service-1079323726684.asia-southeast1.run.app',
```

Comment:
I am not 100% sure but I think we do not need to enable CORS for the question-service because it is a backend service. Do give it a try!

Reply:
You're right, removed! TIL!